docs/source/prereqs.md

```diff
@@ -0,0 +1,232 @@
+# Prerequisites
+
+Let's start with Prerequisites:
+
+
+#### Mac
+
+> **Homebrew**
+> For macOS, we recommend using [Homebrew](https://brew.sh) to manage the prereqs.
+> ```shell
+> $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+> $ brew --version
+> Homebrew 2.5.2
+> ```
+
+> The Xcode command line tools will be installed as part of the Homebrew installation.  Once Homebrew is ready, installing the necessary prerequisites is very easy:
+
+> **Git**
+> Install the latest version of [git](https://git-scm.com/downloads) if it is not already installed.
+
+> ```shell
+> $ brew install git
+> $ git --version
+> git version 2.23.0
+> ```
+
+> **cURL**
+> Install the latest version of [cURL](https://curl.haxx.se/download.html) if it is not already installed.
+> ```shell
+> $ brew install curl
+> $ curl --version
+> curl 7.64.1 (x86_64-apple-darwin19.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.39.2
+> Release-Date: 2019-03-27
+> ```
+
+> **wget**
+> Install the latest version of [wget](https://www.gnu.org/software/wget/) if it is not already installed.
+> ```shell
+> $ brew install wget
+> $ wget -V
+> GNU Wget 1.20.3 built on darwin19.0.0.
+> ```
+
+> **Docker**
+> Install the latest version of [Docker](https://docs.docker.com/get-docker/) if it is not already installed. 
+> ```shell
+> $ brew cask install --appdir="/Applications" docker
+> ```
+> Docker Desktop must be launched to complete the installation so be sure to open the application after installing it:
+> ```shell
+> $ open /Applications/Docker.app
+> ```
+
+> Once installed, confirm the latest versions of both `docker` and `docker-compose` executables were installed.
+> ```shell
+> $ docker --version
+> Docker version 19.03.12, build 48a66213fe
+> $ docker-compose --version
+> docker-compose version 1.27.2, build 18f557f9
+> ```
+
+
+> **GoLang** 
+> Install the latest version of [go](https://golang.org/doc/install) if it is not already installed.
+> ```shell
+> $ brew install go
+> $ go version
+> go1.14.6 darwin/amd64
+>```
+
+> **JQ**
+> Install the latest version of [jq](https://stedolan.github.io/jq/download/) if it is not already installed.
+> ```shell
+> $ brew install jq
+> $ jq --version
+> jq-1.6
+>```
+
+> **SoftHSM**
```

Comment:
Should we say that installing SoftHSM is (Optional). Only required if you plan to play around with HSM?

I think what I am missing in this section is,  are ALL these prereqs required or if you only want to run the test network for example, is there a subset that are required? Is there a baseline of prereqs required and then if you plan to do xxx then you also need to install yyy?

Reply:
Mostly, I think we need all but the SoftHSM prereqs.  We have 7 in total:
**Git, cURL, Docker, GoLang** are needed for the install.  
**Wget**  I remember having to update it on my machine when I was installing Fabric Samples, its needed to download multiple files in your system and I think(??) it's used indirectly by one of the other commands.  
**JQ** is used to update channel configuration when use use `confixtxlator` in our tutorials, like [here ](https://hyperledger-fabric.readthedocs.io/en/latest/channel_update_tutorial.html?highlight=jq#convert-the-configuration-to-json-and-trim-it-down).
**SoftHSM** - You are correct, you only need SoftHSM if you want to use HSM.  I have updated the section with "SoftHSM is Optional - Please install if you plan to play use HSM in your application"